ut/s3_put_chunk_upload_object_action_test.cc

```diff
@@ -953,7 +950,6 @@ TEST_F(S3PutChunkUploadObjectActionTestNoAuth, SaveMetadata) {
   EXPECT_CALL(*(motr_writer_factory->mock_motr_writer), get_oid())
       .Times(AtLeast(1))
       .WillOnce(Return(oid));
-  EXPECT_CALL(*(object_meta_factory->mock_object_metadata), set_oid(_))
```

Comment:
this looks like an error, this should not even compile, as removing this line makes the next one syntax-wrong:

```
.Times(AtLeast(1));
```

Reply:
It's inside commented snippet. Corrected